src/prompts/ui.tsx

```diff
@@ -211,6 +261,13 @@ export default class Ui extends Component<UiProps, UiState> {
         );
     }
 
+    renderUnderlay() {
+        const { prompts } = this.props;
+        const lastPrompt = prompts?.[prompts.length - 1];
```

Comment:
What controls the ordering of prompts?

Appreciate the description says we only show one overlay to prevent stacking them (makes complete sense!), I'm just wondering if you could end up in a situation where you have two configured:

1 - bell no overlay
2 - alert w/ overlay

and due to ordering bell is passed as last prompt and the alert's underlay isn't shown.

If that could happen, would we be better passing say first underlay config of any visible prompt? Not sure.

Reply:
Yeah, it mostly followed the same behaviour as the existing bell - meant to ask about this more, will catch up directly about this with you tho!

Reply:
Updated and opted for showing the first underlay available from visible prompts 👍